Modernize code

eumiro opened the pull request:
1. Remove useless default encoding for source files (UTF-8 is the default)
2. Simplify loops and conditions

sepandhaghighi commented on art/__init__.py:
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Would you please revert this default encoding?
We need this one for some special cases.

sepandhaghighi reviewed (changes requested):
@eumiro Well done 💯 

Please take a look at my comment about **default encoding** and also:

1. Update `CHANGELOG.md`
2. Update `AUTHORS.md`

eumiro commented:
Thank you for the feedback, @sepandhaghighi. Dropped the commit with the `coding` removal and updated `AUTHORS.md`. The other commit is a pure refactoring, so it does not really make sense to add it to `CHANGELOG.md`. It does not change any functionality of the code.

sadrasabouri reviewed (approved):
LGTM

sepandhaghighi reviewed (approved):
🚀
